tdrs-backend/tdpservice/parsers/models.py

```diff
@@ -25,8 +25,8 @@ class Meta:
     item_number = models.IntegerField(null=False)
     field_name = models.TextField(null=False, max_length=128)
     category = models.IntegerField(null=False, default=1)
-    case_number = models.TextField(null=True, max_length=128)
-    rpt_month_year = models.IntegerField(null=True, blank=False)
+    rpt_month_year = models.TextField(null=True, max_length=128)
+    case_number = models.IntegerField(null=True, blank=False)
 
```

Comment:
@raftmsohani It seems to me we overlooked this previously. This should have been flipped, yes?